### PR TITLE
Update json-eval expression to use entire json body.

### PR DESCRIPTION
**Pull request by oprabin**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by Mariangela** [comment]
@arunans23 , please confirm this fix.

**Comment by oprabin** [comment]
Contributor License Agreement is signed.

**Comment by arunans23** [comment]
It seems Integration Studio is not supporting `json-eval($.)` for some reason. Rather than using `$.*`, I prefer just using `$` which will work as well. @oprabin WDYT?

**Comment by oprabin** [comment]
I will test the above and confirm this asap.

File: en/docs/tutorials/integration-tutorials/using-the-gmail-connector.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ You can now start updating the API resource with the mediation flow.
     | New Property Name | hospital_response        |
     | Property Action   | Select **Set**          |
     | Value Type        | Select **Expression**   |
-    | Value Expression  | json-eval($.)           |
+    | Value Expression  | json-eval($.*)           |
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/using-the-gmail-connector.md:325]:
```suggestion
    | Value Expression  | json-eval($)           |
```

**Review by Mariangela — APPROVED** [review]
Updated the doc based on @arunans23 comment.

**Comment by oprabin** [comment]
Hi @Mariangela,
My bad. If that works, feel good to merge it. I won't be able to test thing for a while.

Thanks.

**Pull request merged**